src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachContent.tsx

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import CreditCardIllustration from "../images/high-risk-data-breach-credit-card.svg";
+import BankAccountIllustration from "../images/high-risk-data-breach-bank-account.svg";
+import pinIllustration from "../images/high-risk-data-breach-pin.svg";
+import SocialSecurityNumberIllustration from "../images/high-risk-data-breach-ssn.svg";
+import NoBreachesIllustration from "../images/high-risk-breaches-none.svg";
+import { useL10n } from "../../../../../../../hooks/l10n";
+
+export const HighRiskBreachContent = (typeOfBreach: string) => {
+  const l10n = useL10n();
+  let title,
+    secondaryDescription,
+    recommendationSteps,
+    breachIllustration,
+    illustrationAlt;
+  // TODO: Expose email list & count here https://mozilla-hub.atlassian.net/browse/MNTOR-2112
+  const emailsMonitored = ["email1@gmail.com", "email2@gmail.com"]; // mocked
+  const emailsFormatter = new Intl.ListFormat("en-US", {
```

Comment:
I think there was a previous comment above, but should this use the user's locale here instead of hardcoding "en-US"?
```sh
git grep -Pn "new Intl.\w+\([\"']" | cat

src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachContent.tsx:21:  const emailsFormatter = new Intl.ListFormat("en-US", {
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx:46:  const dateFormatter = new Intl.DateTimeFormat("en-US", {
```

Or gets a _bit_ more interesting if we <kbd>grep</kbd> for "en-US" string:
```sh
git grep "[\"']en-US[\"']" | cat

src/app/(nextjs_migration)/layout.tsx:        ["en", "en-GB", "en-US", "en-CA"].includes(locale) && (
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachContent.tsx:  const emailsFormatter = new Intl.ListFormat("en-US", {
src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx:  const dateFormatter = new Intl.DateTimeFormat("en-US", {
src/app/(proper_react)/redesign/(authenticated)/user/welcome/EnterInfo.tsx:      displayValue: new Date(dateOfBirth).toLocaleDateString("en-US", {
```